proxy/hdrs/MIME.h

```diff
@@ -1316,16 +1313,17 @@ MIMEHdr::value_get_index(const char *name, int name_length, const char *value, i
 inline const char *
 MIMEHdr::value_get(const char *name, int name_length, int *value_length_return) const
 {
-  if (const MIMEField *field = field_find(name, name_length); field) {
-    return field->value_get(value_length_return);
-  }
-  return nullptr;
+  const MIMEField *field = field_find(name, name_length);
+
+  return field ? field->value_get(value_length_return) : nullptr;
 }
 
 inline std::string_view
 MIMEHdr::value_get(std::string_view const &name) const
 {
-  if (MIMEField const *field = field_find(name.data(), name.size()); field) {
+  MIMEField const *field = field_find(name.data(), name.size());
```

Comment:
Oh, this one is avoiding use of C++17 style. I'm fine with either styles, but not ternary operator.

Reply:
I had to do this because there's a bug in cppcheck  on this type of construct, and it made it stop working completely :-/.